webcompat/static/js/lib/wizard/steps/sub-category.js

```diff
@@ -21,8 +21,19 @@ const resetRadio = (element) => {
   });
 };
 
-const handleSubCategory = () =>
+const updateDescription = (target) => {
+  const text = $(target).next("label").text().trim();
+  const toUpdate = {
+    data: { elementId: "#description", value: text },
+    single: true,
+  };
+  notify.publish("updateStep", { id: "hidden", data: toUpdate });
+};
+
+const handleSubCategory = (event) => {
+  updateDescription(event.currentTarget);
```

Comment:
I've noticed that after submitting subcategory is missing in the issue description, so this code is fixing it